lib/index.ts

```diff
@@ -17,7 +17,7 @@ function mapArgument(argument: any, index: number): any {
     : argument
 }
 
-const delegateFnBodyToExecuteInPage = `
+let delegateFnBodyToExecuteInPage = `
```

Comment:
let's save the original as `const delegateFnBody` so we can configure more than once :)